planning/behavior_path_planner/src/behavior_path_planner_node.cpp

```diff
@@ -592,15 +593,83 @@ AvoidanceParameters BehaviorPathPlannerNode::getAvoidanceParam()
 
   // target object
   {
-    std::string ns = "avoidance.target_object.";
-    p.avoid_car = declare_parameter<bool>(ns + "car");
-    p.avoid_truck = declare_parameter<bool>(ns + "truck");
-    p.avoid_bus = declare_parameter<bool>(ns + "bus");
-    p.avoid_trailer = declare_parameter<bool>(ns + "trailer");
-    p.avoid_unknown = declare_parameter<bool>(ns + "unknown");
-    p.avoid_bicycle = declare_parameter<bool>(ns + "bicycle");
-    p.avoid_motorcycle = declare_parameter<bool>(ns + "motorcycle");
-    p.avoid_pedestrian = declare_parameter<bool>(ns + "pedestrian");
+    std::string ns = "avoidance.target_object.motorcycle.";
```

Comment:
@satoshi-ota I love this improvement. Nice. 

Suggest using lambda or function to get rid of duplicated implementations.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/3002/commits/0e172b3532fbf210f540915d7af90839b1bd9c16.